airflow/api/auth/backend/kerberos_auth.py

```diff
@@ -14,13 +14,9 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+"""Kerberos authentication module."""
 from __future__ import annotations
 
-import warnings
```

Comment:
@potiuk , pre-commit hooks reformatted this module after I moved doc string to Line 17 from Line 48. Should I revert this change? 

I thought, instead of merging `E`*s and `D`*s , like you mentioned, I could just fix the conflicts

Reply:
@potiuk I guess this license notice might be removed because it is already listed in https://github.com/apache/airflow/blob/main/3rd-party-licenses/LICENSE-flask-kerberos.txt